docs/scripts/tutorial_nav.js

```diff
@@ -0,0 +1,41 @@
+var navTag = 'h3';
+
+window.addEventListener('DOMContentLoaded', (event) => {
```

Comment:
this whole function builds the navigation DOM programmatically on the client side. is there any reason we chose not to do this in the template itself? (see previous comment).

the `scroll` handler totally only works on the client side 👍